pkg/config/v2/server.go

```diff
@@ -65,6 +65,7 @@ type ListenerConfig struct {
 	StreamFilters         []Filter            `json:"stream_filters,omitempty"`
 	Inspector             bool                `json:"inspector,omitempty"`
 	ConnectionIdleTimeout *api.DurationConfig `json:"connection_idle_timeout,omitempty"`
+	ReadBufferCapacity    int                 `json:"read_buffer_capacity,omitempty"`
```

Comment:
should rename to 'DefaultReadBufferSize'? otherwise it might be difficult to tell apart from 'PerConnBufferLimitBytes'

Reply:
good idea